docker/dev/php/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM php:7.4
+
+COPY envars.sh /
+COPY parameters.yaml.docker /
+COPY entrypoint.sh /
+
+RUN chmod +x /entrypoint.sh
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install \        
+        git \
+        curl \
+        acl \
+        rsnapshot \
+        gzip \
+        zip \
+        mariadb-client \
+        fakeroot \
+        lintian \
+        && rm -rf /var/lib/apt/lists/*
+
+RUN docker-php-ext-install \
+        pdo_mysql \
+        pcntl
+
```

Comment:
In order to be truly useful for development we also need xdebug:
Something like that might do:
```
RUN pecl install xdebug-3.0.3 \
    && docker-php-ext-enable xdebug \
    # Uncomment for troubleshooting XDebug issues (Be aware: very very slow and huge log file)
    #&& echo "xdebug.log=/srv/config/exchange/xdebug.log" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
    #&& echo "xdebug.log_level=7 " >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
    && echo "xdebug.mode=debug" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
    && echo "xdebug.discover_client_host=1" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
    # 172.17.0.1 is the default bridge ip for docker managed networks so pointing to it takes us to the host
    && echo "xdebug.client_host=172.17.0.1" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
```